ovos_date_parser/dates_ca.py

```
@@ -13,6 +14,92 @@
+            time_units[unit_en] += float(match.group(1))
+            return ''
+
+        text = re.sub(unit_pattern, repl, text)
+
+    for (unit_en, unit_ca) in non_std_un.items():
+        unit_pattern = pattern.format(
+            unit=unit_ca[:-1])  # remove 's' from unit
+
+        def repl_non_std(match):
+            val = float(match.group(1))
+            if unit_en == "months":
```

Ruff

Function definition does not bind loop variable `unit_en`

(B023)